pkgs/development/tools/pur/default.nix

```
@@ -4,25 +4,17 @@
 }:
 
 let
-  py = python3.override {
-    packageOverrides = self: super: {
-      # newest version doesn't support click >8.0 https://github.com/alanhamlett/pip-update-requirements/issues/38
-      # Use click 7
-      click = self.callPackage ../../../development/python2-modules/click/default.nix { };
-    };
-  };
-  inherit (py.pkgs) buildPythonApplication click pytestCheckHook;
+  inherit (python3.pkgs) buildPythonApplication click pytestCheckHook;
 in
-
 buildPythonApplication rec {
```

Use `python3.pkgs` and remove the `let...in` as it's no longer needed.

```suggestion
python3.pkgs.buildPythonApplication rec {
```